src/btree/bt_cursor.c

```diff
@@ -558,15 +588,18 @@ __wt_btcur_search_near(WT_CURSOR_BTREE *cbt, int *exactp)
 			exact = -1;
 	}
 
+err:	if (exactp != NULL && (ret == 0 || ret == WT_NOTFOUND))
+		*exactp = exact;
```

Comment:
Can you please add a comment explaining why we need to set `exactp` when `ret == WT_NOTFOUND`?